### DISCUSSION TITLE
Form with fields from related table

### DISCUSSION THREAD

**Discussion by italo1983** [directus/directus#18003] [Archive]
Hello,
I need to build a form with fields shared between multiple collections, like Redmine where each Tracker (a form) show only a set of fields configured in custom_fields_tracker table and fields values are saved on a custom_values table containing issue_id, field_id and the value.
How can I achieve this in directus? There's something that let me share fields between forms without creating long tables?

Thank you

**Comment by xu4wang** [comment]
You can create a module extension for this.

Using the v-form component, or several v-form components on your form page to collect user inputs, and making a call to your own API endpoint to save.
